infra/frontend/app-config/prod.tf

```diff
@@ -19,4 +19,9 @@ module "prod_config" {
 
   instance_cpu    = 1024
   instance_memory = 2048
+
+  service_override_extra_environment_variables = {
+    NEW_RELIC_ENABLED = "true"
+    NODE_OPTIONS      = "-r @newrelic/next"
```

Comment:
RE `NODE_OPTIONS`, can we just always input `-r @newrelic/next` and rely on the `NEW_RELIC_ENABLED` flag instead? It's gonna be unclear to people looking at the infra layer what they should be doing with `NODE_OPTIONS`

Reply:
yeah, I'm fine with that. In the end we'll have this enabled everywhere, and pre-loading the New Relic library, even if we aren't using it, shouldn't pose any problems.